tools/serverpod_cli/lib/src/util/string_manipulation.dart

```diff
@@ -20,6 +27,26 @@ List<String> splitIgnoringBrackets(
         depth++;
       } else if (char == '>' || char == ')') {
         depth--;
+      } else if (char == "'" && !insideDoubleQuotes) {
+        if (insideSingleQuotes) {
+          // Check if it's an escaped single quote
+          if (i > 0 && input[i - 1] == '\\') {
+            continue;
+          }
+          insideSingleQuotes = false;
+        } else {
+          insideSingleQuotes = true;
+        }
+      } else if (char == '"' && !insideSingleQuotes) {
```

Comment:
Question: Why don't we add and remove depth here to have the solution in line with the other parsing?